frontend/views/pages/DesignSystem.vue

```diff
@@ -403,13 +430,30 @@ page(pageTestName='designSystemPage' pageTestHeaderName='designSystemTitle' clas
       h2 Tooltips
 
     section.p-section
+      tooltip(
+        text="A simple text inside"
+        )
+        i.icon-exclamation-triangle Basic
+
+      | &nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
@sandrina-p @pieer is there a better solution, maybe with padding or margin than these `&nbsp;`s? They seem a bit hackish...

Reply:
Oh yes, we could use paddings, but this is the DesignSystem file, like our "playground", so I didn't think it was really needed just for showcase.